tests/integration/test_select.py

```diff
@@ -222,3 +222,13 @@ def test_case_when_no_else(c):
     expected_df = pd.DataFrame({"C": [None, 1, 1, 1, None]})
 
     assert_eq(actual_df, expected_df)
+
+
+def test_singular_column_projection_simple(c):
+    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+    c.create_table("df", df)
+
+    wildcard_result = c.sql("SELECT * from df").compute()
+    single_col_result = c.sql("SELECT b from df").compute()
```

Comment:
We shouldn't need to `compute` here, and generally it's better to avoid whenever possible so we get some extra testing on the Dask collection itself:

```suggestion
    wildcard_result = c.sql("SELECT * from df")
    single_col_result = c.sql("SELECT b from df")
```